src/ui/pages/add/PagesAddFormContainer.js

```diff
@@ -15,10 +15,29 @@ import { PAGE_INIT_VALUES, SEO_DATA_BLANK, SEO_LANGDATA_BLANK } from 'ui/pages/c
 import { getLocale } from 'state/locale/selectors';
 import getSearchParam from 'helpers/getSearchParam';
 import { setVisibleModal } from 'state/modal/actions';
-import { getAppTourProgress, getTourCreatedPage } from 'state/app-tour/selectors';
+import { getAppTourProgress, getTourCreatedPage, getExistingPages } from 'state/app-tour/selectors';
 import { APP_TOUR_STARTED } from 'state/app-tour/const';
 import { setAppTourLastStep, setTourCreatedPage } from 'state/app-tour/actions';
 
+export const getNextIncremetalPropertyName = (pages, property, keyword, extraChar) => {
```

Comment:
can you please make it more functional?

Reply:
I'll made some commits to show up what I mean (and also something to improve test readability)